unlock-app/src/stories/creator/DemoPage.stories.js

```diff
@@ -4,28 +4,37 @@ import { storiesOf } from '@storybook/react'
 import Demo from '../../pages/demo'
 import createUnlockStore from '../../createUnlockStore'
 
+const myLock = {
+  address: '0xaaaaaaaaa0c4d48d1bdad5dcb26153fc8780f83e',
+  name: 'My Blog',
+  keyPrice: '27000000000000000',
+  expirationDuration: '172800',
+  maxNumberOfKeys: 240,
+  outstandingKeys: 3,
+}
 const store = createUnlockStore({
-  locks: {
-    '0x56781234a': {
-      address: '0x56781234a',
+  locks: [
```

Comment:
Hum... I think this is odd, IIRC in our redux stores, `locks` are an object, not array.

Reply:
very odd, but I was getting all kinds of warnings in the console and this fixed it. Does our mapStateToProps convert to an array perhaps? I'll investigate, because i didn't change any source code to make these stories work :)

Reply:
I had more time to investigate, and you're right (of course), so the commits I just pushed should resolve these issues. There is one other nasty looking warning in the console I need to investigate, so I'll mark it WIP until I figure it out